### ISSUE TITLE
Research how to implement rights changes in pywb

### ISSUE THREAD

**Issue by edsu** [sul-dlss/was-pywb#5]
We have a requirement from @apchan to support changing access rights to archived web content (world, stanford, dark). Currently changes to seed object rights in Argo only controls how things appear in SearchWorks and PURL, and has no effect on the visibility of content in SWAP (currently OpenWayback). Crawl objects are all dark in SDR, which has no effect on swap, which makes their contents all available to the public.

It is possible to use [blocks, exclusions and user based access controls](https://pywb.readthedocs.io/en/embargo/manual/access-control.html#access-control-settings) in pywb. But it is unclear what work would need to be done to integrate changes in Argo with changes in the pywb configuration. This task is to come up with a design for that.

**Comment by ndushay** [comment]
This is now unblocked;  I'm moving it to ready.

**Comment by jcoyne** [comment]
@edsu if the object is dark, then it should be removed from PyWB entirely, right? That is what is done in stacks.

**Comment by lwrubel** [comment]
Adding to the initial description: in talking with IIPC pywb users, pywb does not have an API endpoint for making these kinds of changes. It seems most other users are not programmatically making these changes; they are editing the files. Another library is working on creating a Python script to generate .aclj from a flat file of exclusions. (I don't think it's public but let me know and I can share it)

Note that changes are not picked up automatically when added. We may want to reload config files every few hours/daily like UKWA is doing. (Or @edsu may be thinking of a way to do this.)

**Comment by edsu** [comment]
@jcoyne at the moment removing a record that matches a URL would involve rewriting the WARC data so that the records containing that URL and any of the page's constituent resource representations are removed. I think that @andrewjbtw @apchan decided this was out of scope for the current work cycle, and that preventing it from being viewed would be acceptable. This is what we are currently doing manually with our `exclusion.txt`.

**Comment by jcoyne** [comment]
@andrewjbtw  Doesn't dark mean we totally disavow the record, rather than say "you aren't authorized to see this record"? Is this still an important requirement?

**Comment by andrewjbtw** [comment]
No, dark just means we're preserving it but restricting it from public view. Decommission is when we say it's not an active record.

**Comment by edsu** [comment]
Yes, thanks for the feedback, and my apologies for not taking it earlier. Here's the more actionable bit of work that has resulted from this:

https://github.com/sul-dlss/was-pywb/issues/10
